recipe/patches/0013-Add-bazel-linkopts-libs.patch

```diff
@@ -0,0 +1,25 @@
+From 00917f6519deca3b8db735a92f0c7d5ee5a7d44e Mon Sep 17 00:00:00 2001
+From: Kai Fricke <kai@anyscale.com>
+Date: Fri, 6 Aug 2021 12:36:05 +0100
+Subject: [PATCH 13/13] Add bazel linkopts/libs
+
+---
+ python/setup.py | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/python/setup.py b/python/setup.py
+index c7964b399..5d6bc93e3 100644
+--- a/python/setup.py
++++ b/python/setup.py
+@@ -269,6 +269,8 @@ def build(build_python, build_java):
+         raise OSError(msg)
+ 
+     bazel_env = dict(os.environ, PYTHON3_BIN_PATH=sys.executable)
++    bazel_env.setdefault("BAZEL_LINKOPTS", "-static-libstdc++:-lm")
++    bazel_env.setdefault("BAZEL_LINKLIBS", "-l%:libstdc++.a")
```

Comment:
why do we need this?

Reply:
Local build wouldn't pass otherwise (getting linker errors in grpc/protobuf builds)